install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -428,10 +428,10 @@ project that has a claim to them.
 - Configure xref:nfs-reclaiming-resources[reclamation of discarded resources] for
 each PV.
 
-They are many ways that you can use scripts to automate the above tasks. You can
+They are many ways that you can use scripts to automate the preceding tasks. You can
```

Comment:
I suggest changing "They" to "There". I.e:

"There are many ways that you can use scripts to automate the preceding tasks."